sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -2516,4 +2516,26 @@ class HiveDDLSuite
       }
     }
   }
+
+  test("SPARK-28313: Spark sql null type incompatible with hive void type") {
+    val expectedMsg = "DataType void is not supported"
+    withTable("t") {
+      val e = intercept[AnalysisException] {
+        sql("create table t (a void)")
```

Comment:
how is this related to hive compatibility? It seems to me that you just change this statement from parser exception to analysis exception.

Reply:
This code is just to confirm spark cannot do this ddl that compatible with hive.